android/modules/ui/src/java/ti/modules/titanium/ui/NavigationWindowProxy.java

```diff
@@ -59,21 +62,32 @@ public void close(@Kroll.argument(optional = true) Object arg)
 	// clang-format on
 	{
 		popToRootWindow(arg);
-		closeWindow(windows.get(0), arg); // close the root window
+		closeWindow(((WindowProxy) windows.get(0)), arg); // close the root window
```

Comment:
Same issue mentioned above.
This will crash if `windows` member variable returns a `TabGroupProxy` object.